eZ/Publish/Core/settings/storage_engines/legacy/content.yml

```
@@ -9,6 +9,16 @@ services:
         arguments:
             - "@ezpublish.persistence.legacy.field_value_converter.registry"
             - "@ezpublish.spi.persistence.legacy.language.handler"
+            - '@ezpublish.spi.persistence.legacy.content_type.handler'
+            - '@Symfony\Contracts\EventDispatcher\EventDispatcherInterface'
+
+    Ibexa\Core\Persistence\Legacy\Content\Mapper\ResolveVirtualFieldSubscriber:
+        arguments:
+            - "@ezpublish.persistence.legacy.field_value_converter.registry"
+            - "@ezpublish.persistence.external_storage_registry"
+            - "@ezpublish.persistence.legacy.content.gateway"
```

Inconsistent quoting, also please use named arguments in new DIC, it's easier to maintain.
```suggestion
            $converterRegistry: '@ezpublish.persistence.legacy.field_value_converter.registry'
            $storageRegistry: '@ezpublish.persistence.external_storage_registry'
            $contentGateway: '@ezpublish.persistence.legacy.content.gateway'
```